bin/benchmark-servers-stop.sh

```diff
@@ -82,10 +82,10 @@ for host_name in "${hosts0[@]}";
 do
     if [[ ${host_name} = "127.0.0.1" || ${host_name} = "localhost" ]]
         then
-            pkill -9 -f "Dyardstick.server"
+            pkill -f "Dyardstick.server"
 
         else
-            `ssh -o PasswordAuthentication=no ${REMOTE_USER}"@"${host_name} pkill -9 -f "Dyardstick.server"`
```

Comment:
@SviridovMM if you have promlems with JFR then just set duration in profiler parameters (see below) less than benchmarks duration.

> -XX:StartFlightRecording=duration=Ns


Changing the signal for the pkill command can cause problems with hung Java processes.

Reply:
@isuntsov-gridgain, could you explain why this change may lead to hanging?